### PR TITLE
Make error pages translatable

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -81,6 +81,10 @@ loading = Loading…
 
 error404 = The page you are trying to reach either <strong>does not exist</strong> or <strong>you are not authorized</strong> to view it.
 
+[error]
+occurred = An error has occurred :
```
Comment (6543) [inline options/locale/locale_en-US.ini]:
```suggestion
occurred = An error has occurred
```

File: templates/status/500.tmpl [hunk 2]
```diff
@@ -3,9 +3,9 @@
 	<p style="margin-top: 100px"><img class="ui centered image" src="{{StaticUrlPrefix}}/img/500.png" alt="500"/></p>
 	<div class="ui divider"></div>
 	<br>
-	{{if .ErrorMsg}}<p>An error has occurred :</p>
+	{{if .ErrorMsg}}<p>{{.i18n.Tr "error.occurred"}}</p>
```
Comment (6543) [inline templates/status/500.tmpl]:
```suggestion
	{{if .ErrorMsg}}<p>{{.i18n.Tr "error.occurred"}}:</p>
```

**Review by zeripath — APPROVED** [review]
I'm not certain this really gets us very fast but I guess it's a start

File: templates/status/500.tmpl [hunk 3]
```diff
@@ -3,9 +3,9 @@
 	<p style="margin-top: 100px"><img class="ui centered image" src="{{StaticUrlPrefix}}/img/500.png" alt="500"/></p>
 	<div class="ui divider"></div>
 	<br>
-	{{if .ErrorMsg}}<p>An error has occurred :</p>
+	{{if .ErrorMsg}}<p>{{.i18n.Tr "error.occurred"}}:</p>
 	<pre style="text-align: left">{{.ErrorMsg}}</pre>{{end}}
-	{{if .ShowFooterVersion}}<p>Application Version: {{AppVer}}</p>{{end}}
-	{{if .IsAdmin}}<p>If you are sure this is Gitea bug, please search for issue on <a href="https://github.com/go-gitea/gitea/issues">GitHub</a> and open new issue if necessary.</p>{{end}}
+	{{if .ShowFooterVersion}}<p>{{.i18n.Tr "admin.config.app_ver"}}: {{AppVer}}</p>{{end}}
+	{{if .IsAdmin}}<p>{{.i18n.Tr "error.report_message"}}</p>{{end}}
```
Comment (jaqra) [inline templates/status/500.tmpl]:
```suggestion
	{{if .IsAdmin}}<p>{{.i18n.Tr "error.report_message" | Safe}}</p>{{end}}
```

**Review by ghost — COMMENTED** [review]
This is needed?

File: templates/status/500.tmpl (hunk 3 above)
Comment (6543) [reply templates/status/500.tmpl]: thought about that already - but is the link to the bug tracker klickable afterwards?

Comment (guillep2k) [reply templates/status/500.tmpl]: I think it's the opposite: `Safe` will mark the text as safe and _skip_ sanitization. 🤔 
https://golang.org/pkg/html/template/#HTML

**Comment by 6543** [comment]
@jaqra tested yes It need Safe

**Comment by davydov-vyacheslav** [comment]
@6543 Ain't I correct that error 500 should occur when server got exceptional cases? If so, it would be helpful, even to create a ticket, to have a stacktrace of the exception, wouldn't it?

**Comment by 6543** [comment]
You will have if you set the gitea run mode to dev @davydov-vyacheslav

**Comment by 6543** [comment]
need one review

**Review by ghost — APPROVED** [review]

**Pull request merged**